salesagility/SuiteCRM
# Fix #9476 - Mass assign security groups only assigns selected on current page

Change how UID array is built in assignGroups



## Description
Use MassUpdate.uid to build ID array then set MassAssign to be equal to MassUpdate array





## Motivation and Context
Fix issue where mass assign security groups only assigns selected on current page


## How To Test This
1. Select 10 records on page 1, go to next page of records and select 10 more records, now 20 in total.
2. Mass assign a security group
3. Check securitygroups_record table to ensure all records were assigned a group


## Types of changes


### Final checklist
